var/spack/repos/builtin/packages/py-parasail/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyParasail(PythonPackage):
+    """Python Bindings for the Parasail C Library. Parasail is a SIMD C (C99)
+    library containing implementations of the Smith-Waterman (local),
+    Needleman-Wunsch (global), and semi-global pairwise sequence alignment
+    algorithms."""
+
+    homepage = "https://github.com/jeffdaily/parasail-python"
+    pypi = "parasail/parasail-1.3.3.tar.gz"
+
+    version("1.3.3", sha256="06f05066d9cf624c0b043f51a1e9d2964154e1edd0f9843e0838f32073e576f8")
+
```

Comment:
Prob also want to add perl and m4: https://github.com/jeffdaily/parasail-python/blob/v1.3.3/setup.py#L164